pkg/kubectl/cmd/annotate.go

```diff
@@ -31,6 +31,7 @@ import (
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/kubectl/resource"
 	"k8s.io/kubernetes/pkg/runtime"
+	. "k8s.io/kubernetes/pkg/util/i18n"
```

Comment:
I would prefer not to do this, `i18n.T` is short and it plays much better with tools.